src/sentry/models/activity.py

```diff
@@ -32,7 +32,7 @@
 
 
 class ActivityManager(BaseManager["Activity"]):
-    def get_activities_for_group(self, group: Group, num: int) -> Sequence[Group]:
```

Comment:
Am I crazy or should this be returning Activities and not Groups?

Reply:
Yeah it should! not sure why it was typed as groups 🤔